_laptop/thinkpad_x13s.md

```diff
@@ -0,0 +1,131 @@
+---
+name: ThinkpadX13s
+layout: laptop
+status-audio: N/A
+status-audio-displayportaudio: N
+status-audio-mic: 6.5
+status-audio-mic-comment: "
+    Some capture issues
+"
+status-audio-speakers: 6.5
+status-audio-speakers-comment: "
```

Comment:
I like how the comment has double use on the inividual davice page and as tooltip on the main matrix. Nice! :-)

Reply:
yeah, not to have the credits, the framework works like that by default. ehehe

Reply:
Fair enough, keep the credits though :-)